src/command/info/server.ts

```diff
@@ -3,6 +3,7 @@ import { Message } from 'discord.js';
 import { MessageEmbed } from 'discord.js';
 import { prettyDate } from '../../util/functions';
 import { startCase } from 'lodash';
+import { truncate } from 'lodash';
```

Comment:
Please use one combined import statement for both lodash methods

Reply:
whoops, my bad :+1: we all good now